Explorer/Assets/DCL/Backpack/BackpackCharacterPreviewView.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace DCL.CharacterPreview
+{
+    public class BackpackCharacterPreviewView : MonoBehaviour, IDisposable
+    {
+        [field: SerializeField]
+        public RawImage RawImage { get; private set; }
+
+        [field: SerializeField]
+        public CharacterPreviewContainer CharacterPreviewContainer { get; private set;}
```

Comment:
It should not belong here: `CharacterPreviewContainer` should be a completely detachable prefab with an encapsulated scene. What can be here is some settings that you can pass as an argument to the Factory (if needed)